src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -524,6 +558,8 @@ void CoilCoolingDXCurveFitPerformance::calcStandardRatings210240(EnergyPlus::Ene
             CurveManager::CurveValue(state, speed.indexCapFT, CoolingCoilInletAirWetBulbTempRated, OutdoorUnitInletAirDryBulbTempRated);
         this->standardRatingCoolingCapacity =
             mode.ratedGrossTotalCap * TotCapTempModFac * TotCapFlowModFac - FanPowerPerEvapAirFlowRate * mode.ratedEvapAirFlowRate;
+        this->standardRatingCoolingCapacity2023 =
```

Comment:
Calculations littered through here to mimic the older rating calculations for the new values.  Although it's possible we could collect these together into a worker function, there may be some subtle differences that would make it a bit difficult.  Anyway, that can be done later, this seems fine.